src/blockrelay/graphene.h

```diff
@@ -123,10 +124,9 @@ class CGrapheneBlock
 
     // Validates header and, if possible, determines if there are any missing or unnecessary transactions
     // in the block
-    bool ValidateAndRecontructBlock(int &missingCount,
-        int &unnecessaryCount,
-        uint256 blockhash,
+    bool ValidateAndRecontructBlock(uint256 blockhash,
         std::shared_ptr<CBlockThinRelay> pblock,
+        std::map<uint64_t, CTransactionRef> mapMissingTx,
```

Comment:
use const std::map<uint64_t, CTransactionRef>& mapMissingTx, unless you want to make a copy of the map.

Reply:
Changed to const.